extensions/gcp/processors/ListGCSBucket.h

```diff
@@ -21,12 +21,26 @@
 #include <string>
 #include <utility>
 
+#include "../GCPAttributes.h"
 #include "GCSProcessor.h"
 #include "core/logging/LoggerConfiguration.h"
+#include "core/OutputAttributeDefinition.h"
+#include "core/PropertyDefinition.h"
+#include "core/PropertyDefinitionBuilder.h"
+#include "core/PropertyType.h"
+#include "core/RelationshipDefinition.h"
 #include "utils/ArrayUtils.h"
 
 namespace org::apache::nifi::minifi::extensions::gcp {
 
+namespace detail {
+inline constexpr std::string_view FILENAME_OUTPUT_ATTRIBUTE_DESCRIPTION_PART_1{"Same as "};
+inline constexpr auto FILENAME_OUTPUT_ATTRIBUTE_DESCRIPTION_ARRAY = utils::array_cat(
+    utils::string_view_to_array<FILENAME_OUTPUT_ATTRIBUTE_DESCRIPTION_PART_1.size()>(FILENAME_OUTPUT_ATTRIBUTE_DESCRIPTION_PART_1),
+    utils::string_view_to_array<GCS_OBJECT_NAME_ATTR.size()>(GCS_OBJECT_NAME_ATTR));
+inline constexpr auto FILENAME_OUTPUT_ATTRIBUTE_DESCRIPTION = utils::array_to_string_view(FILENAME_OUTPUT_ATTRIBUTE_DESCRIPTION_ARRAY);
```

Comment:
All caps are better reserved for macros IMO, I'd use something else for typed constants. I don't think we have a consistent naming scheme for this, but both CamelCase and snake_case works for me. I think CamelCase is a bit more common in the codebase for constants, especially now that they're used for property definitions as well.
The Google C++ style guide recommends kPrefixedCamelCase, but I don't really like that, and we don't follow this anywhere at the moment. https://google.github.io/styleguide/cppguide.html#Constant_Names

Reply:
I'm not sure about this, since we do use ALL_CAPS for many typed constants (e.g. `GCPAttributes.h`); although some of these probably used to be macros.  I think ALL_CAPS could be extended from just macros to compile-time constant numbers and strings.